docs/dev/demo.md

```diff
@@ -0,0 +1,603 @@
+
+...
+
+nautobot-netbox-importer-nautobot-1  |   Applying virtualization.0026_change_virtualmachine_primary_ip_fields... OK
+
+...
+
+nautobot-netbox-importer-nautobot-1  | Starting development server at http://0.0.0.0:8080/
+nautobot-netbox-importer-nautobot-1  | Quit the server with CONTROL-C.
+nautobot-netbox-importer-nautobot-1  | 09:46:22.951 INFO    nautobot             __init__.py                              setup() :
+nautobot-netbox-importer-nautobot-1  |   Nautobot initialized!
+```
+
+After Nautobot is installed and operational, the web interface is accessible at `http://localhost:8080/` using the default username and password: `admin`.
+
+## Verifying Importer Options
+
+To review available options for the importer, execute:
+
+```bash
+invoke import-netbox --help
+
+Usage: inv[oke] [--core-opts] import-netbox [--options] [other tasks here ...]
+
+Docstring:
+  Import NetBox data into Nautobot.
+
+Options:
+  -b, --bypass-data-validation             Bypass as much of Nautobot's internal data validation logic as possible, allowing the import of data from NetBox that would be rejected as invalid if entered as-is through the GUI or REST API. USE WITH CAUTION: it is generally more desirable to *take note* of any data validation errors, *correct* the invalid data in NetBox, and *re-import* with the corrected data! (default: False)
+  -d STRING, --demo-version=STRING         Version of the demo data to import from public NetBox repository (default: empty).
+  -f STRING, --file=STRING                 Path to the JSON file to import.
+  -i, --[no-]fields-mapping                Show a mapping of NetBox fields to Nautobot fields. Only printed when `--summary` is also specified. (default: True)
+  -p, --update-paths                       Call management command `trace_paths` to update paths after the import. (default: False)
+  -r, --[no-]dry-run                       Do not write any data to the database. (default: False)
+  -s STRING, --save-mappings-file=STRING   File path to write the JSON mapping to. (default: generated-mappings.json)
+  -t, --sitegroup-parent-always-region     When importing `dcim.sitegroup` to `dcim.locationtype`, always set the parent of a site group, to be a `Region` location type. This is a workaround to fix validation issues `'A Location of type Location may only have a Location of the same type as its parent.'`. (default: False)
+  -u, --[no-]summary                       Show a summary of the import. (default: True)
+  -x, --fix-powerfeed-locations            Fix panel location to match rack location based on powerfeed. (default: False)
+```
+
+This command displays usage information, a brief description, and a list of command-line options for the `import-netbox` invoke task.
+
+## Start the Importer
+
+To import public NetBox demo data into your local Nautobot instance, run the following command:
+
+```bash
+invoke import-netbox \
+  --demo-version 3.6 \
```

Comment:
why "demo"? what if I'm importing "actual" data?
could I know which versions are available?

Reply:
ok, I get it. The demo-version is for testing, then we should make it more evident as an alternative to the `--file` option that we would use in a real environment

Reply:
you could alos mention where thi demo data is coming from

Reply:
Better description added.